lib/matplotlib/patches.py

```diff
@@ -4059,7 +4059,9 @@ def get_bbox(self):
 
 class FancyArrowPatch(Patch):
     """
-    A fancy arrow patch. It draws an arrow using the `ArrowStyle`.
+    A fancy arrow patch. It draws an arrow using the `ArrowStyle`. It is primarily used
+    by the `~axes.Axes.annotate` method.  For most purposes, use the annotate method 
+    for drawing arrows.
```

Comment:
The first paragraph should be a single line:

```suggestion
    A fancy arrow patch.

    It draws an arrow using the `ArrowStyle`. It is primarily used by the
    `~.axes.Axes.annotate` method.  For most purposes, use the annotate method for
    drawing arrows.
```